hypertrace-graphql-labels-schema-impl/src/main/java/org/hypertrace/graphql/label/deserialization/LabelApplicationRuleDeserializationConfig.java

```diff
@@ -146,6 +146,12 @@ private static class StringConditionArgument implements StringCondition {
 
     @JsonProperty(VALUE_KEY)
     String value;
+
+    @JsonProperty(VALUES_KEY)
```

Comment:
I guess it's pre-existing, but why did this class get implemented twice? Seems like a mistake here.

Reply:
`private static class LabelApplicationRuleArgument` is missing in one of them. Should I remove one of them, the one without `LabelApplicationRuleArgument` class?